package-lock.json

```diff
@@ -0,0 +1,1984 @@
+{
```

Comment:
i don't think we need package-lock.json, could you remove this file?

Reply:
Will remove this 👍🏽